main.js

```diff
@@ -1163,25 +1163,16 @@ function createWindow() {
                 }, 1000)
             })
 
-            let storeMiniplayerSizeTimer
             miniplayer.on('resize', function (e) {
-                try {
-                    let size = miniplayer.getSize()
-                    if (storeMiniplayerSizeTimer) {
-                        clearTimeout(storeMiniplayerSizeTimer)
-                    }
-                    storeMiniplayerSizeTimer = setTimeout(() => {
-                        settingsProvider.set(
-                            'settings-miniplayer-size',
-                            Math.min(...size)
-                        )
-                        miniplayer.setSize(Math.min(...size), Math.min(...size))
-                    }, 500)
-                } catch {
-                    writeLog({ type: 'warn', data: 'error miniplayer resize' })
-                }
-            })
+                // This is a much better solution since it doesn't leave the gray background when you're still dragging it,
+                // Its still a bit glitchly though visually, but hopefully this will do for now
+                // https://github.com/electron/electron/issues/8036#issuecomment-287261029
 
+                let size = miniplayer.getSize()
+                miniplayer.setSize(size[0], parseInt((size[0] * 1) / 1))
```

Comment:
This has been done in commit 8a03ea7. Which one is better?

Reply:
I'll be honest I blindly replaced the code, and it just worked so I assumed it was fine. 
I'll check it out in a bit.

Reply:
I also realized this presents a new problem on some computers, where if you just drag around the window it will resize as you move it. God I'm dumb

https://user-images.githubusercontent.com/18307183/107899380-f49cf480-6ef2-11eb-8cd7-2c73061a9681.mp4

Reply:
Umm.. This video was recorded on current the code of development branch or your implementation?

Reply:
My implementation